restli-common/src/main/java/com/linkedin/restli/internal/common/URIParamUtils.java

```diff
@@ -486,6 +512,7 @@ public static URI replaceQueryParam(URI uri, String queryParam, DataComplex valu
    */
   public static void addSortedParams(UriBuilder uriBuilder, DataMap params)
```

Comment:
Once again, this method is public so you should be careful about changing behavior.

Reply:
yes, it is resolved